lib/models/offering_wrapper.dart

```diff
@@ -51,3 +51,12 @@ class Offering with _$Offering {
   factory Offering.fromJson(Map<String, dynamic> json) =>
       _$OfferingFromJson(json);
 }
+
```

Comment:
Let's add a comment about why this is needed, something along the lines of 
```
Extension needed because this was a deprecation from freezed 1.x that was removed in 2.x?
```